mod-director-core/src/main/java/net/jan/moddirector/core/configuration/type/CurseRemoteMod.java

```diff
@@ -94,4 +105,14 @@ public RemoteModInformation queryInformation() throws ModDirectorException {
         @JsonProperty
         private String[] gameVersion;
     }
+
+	@Override
+	public String folderName() {
+		return this.folderName;
+	}
+	
+	@Override
+	public int forceInject() {
```

Comment:
Instead of adding these here, add them into `ModDirectorRemoteMod`, as they apply to all mod types.